### PR TITLE
Request to release OpenTelemetry.Instrumentation.AWS package

### PR DESCRIPTION
**Pull request by normj**
## Changes
Requesting to release OpenTelemetry.Instrumentation.AWS. I need the change to continue some experimenting work I'm trying to do with the new .NET Aspire project with AWS and the AWS .NET SDK.

@open-telemetry/dotnet-contrib-maintainers

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] Appropriate `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: src/OpenTelemetry.Instrumentation.AWS/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.1.1-beta.1
```
Comment (utpilla) [inline src/OpenTelemetry.Instrumentation.AWS/CHANGELOG.md:5]: This package has not had a stable `1.1.0` release yet.

```suggestion
## 1.1.0-beta.2
```

Comment (normj) [reply src/OpenTelemetry.Instrumentation.AWS/CHANGELOG.md]: Fixed

File: src/OpenTelemetry.Instrumentation.AWS/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.1.0-beta.2
+
+Released 2023-Dec-04
```
Comment (utpilla) [inline src/OpenTelemetry.Instrumentation.AWS/CHANGELOG.md:7]:
```suggestion
Released 2023-Dec-01
```

**Review by utpilla — APPROVED** [review]

**Comment by utpilla** [comment]
Here is the published package: https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AWS/1.1.0-beta.2

**Pull request merged**